tests/merge/end_to_end.rs

```diff
@@ -157,3 +153,66 @@ pub fn test_end_to_end() {
     let res = core.run(tests).expect("tests failed");
     assert_eq!(res.status(), hyper::StatusCode::Ok);
 }
+
+pub fn test_rejection() {
```

Comment:
You should coach people on how to handle rejection.

Reply:
they can handle it with much fewer lines with a framework (I'll show myself to the door)

Reply:
D: